ptscontrol.py

```diff
@@ -748,8 +748,35 @@ def save_test_history_log(self, save):
 
     def get_bluetooth_address(self):
         """Returns PTS bluetooth address string"""
+        try:
+            address = self._pts.GetPTSBluetoothAddress()
+        except Exception as e:
+            address = ""
 
-        return self._pts.GetPTSBluetoothAddress()
+        if address == "":
+            self.connect_to_dongle()
+            try:
+                address = self._pts.GetPTSBluetoothAddress()
+            except Exception as e:
```

Comment:
Looks like this will silently handled, maybe we should raise this exception instead if we failed to obtain the address?